packages/data-portal-explore/src/components/FileTable.tsx

```diff
@@ -866,12 +866,26 @@ export class FileTable extends React.Component<IFileTableProps> {
                             )
                         );
                     } else {
-                        columns.push({
-                            name: selectorToColumnName(selector),
-                            selector,
-                            wrap: true,
-                            sortable: true,
-                        });
+                        if (selector === 'therapy') {
+                            columns.push({
+                                name: 'Treatment Type',
+                                selector: (file: Entity) => {
+                                    return _.uniq(
+                                        file.therapy.map((d) => d.TreatmentType)
+                                    ).join(', ');
+                                },
+                                cell: truncatedTableCell,
+                                wrap: true,
+                                sortable: true,
+                            });
+                        } else {
+                            columns.push({
+                                name: selectorToColumnName(selector),
+                                selector,
+                                wrap: true,
+                                sortable: true,
+                            });
+                        }
```

Comment:
This is a bit confusing.

Can we actually keep this part as is and add `theraphy` to `excludeFromOtherColumns` instead? Then we can add `Treatment Type` as a default column and we can still hide it by default.

Reply:
@onursumer I did try that first, but then in the columns, we don't get a `treatment type` option it only displays `Therapy` :
<img width="965" alt="image" src="https://github.com/ncihtan/htan-portal/assets/71656941/4f12ade5-ffca-45cb-afe3-711aaca3dd57">


which when selected or clicked throws this error:
<img width="965" alt="image" src="https://github.com/ncihtan/htan-portal/assets/71656941/d1313b65-904f-498c-8d47-741729f39849">